debug/index.html

```diff
@@ -239,6 +239,7 @@ <h1 class="title">Transmux Analyzer</h1>
       codecsArray = codecs.value.split(',');
 
       mediaSource.addEventListener('sourceopen', function () {
+        mediaSource.duration = 0;
```

Comment:
After the append, the duration will be updated by MSE if it is greater than the existing duration.